python/doc/examples/data_analysis/statistical_tests/plot_kolmogorov_statistics.py

```diff
@@ -2,57 +2,75 @@
 Kolmogorov-Smirnov : understand the statistics
 ==============================================
 """
-# %% 
+# %%
 
 # %%
-# In this example, we illustrate how the Kolmogorov-Smirnov statistics is computed. 
+# In this example, we illustrate how the Kolmogorov-Smirnov statistics is computed.
 #
-# * We generate a sample from a gaussian distribution. 
+# * We generate a sample from a gaussian distribution.
```

Comment:
```suggestion
# * We generate a sample from a normal distribution.
```

Reply:
I remember that we discussed this point, but unfortunately I do not remember the conclusion. I searched for the difference, but found none: https://stats.stackexchange.com/questions/55962/what-is-the-difference-between-a-normal-and-a-gaussian-distribution I guess that it might be easier for me if that convention was written somewhere in the doc. Anyway : 1) normal is 1D, 2) Gaussian is multidimensional. Is this the convention we agreed on?

Reply:
I don't remember such a discussion between normal and Gaussian. BTW the choice has already been done in the library: it is the Normal distribution. And as the same class implements the 1D and the nD case, I don't see the point of making a difference in the name for the documentation...

Reply:
@mbaudin47 refers to a discussion we had with @adutfoy prior to the work @KieranDelamotte did on the doc. The convention we agreed on was:
- _normal_  (with lower case n) distribution regardless of the dimension: this fits the API where this mathematical notion is represented by the `Normal` class
- _Gaussian_ (with upper case G) random variable or vector (again, regardless of the dimension). I think this fits the standard usage.

Reply:
So, in short, you either sample _a Gaussian vector_ or _from a normal distribution_.